Formula/confluent-platform.rb

```diff
@@ -12,13 +12,32 @@ class ConfluentPlatform < Formula
   conflicts_with "kafka", :because => "kafka also ships with identically named Kafka related executables"
 
   def install
-    prefix.install "bin"
-    rm_rf "#{bin}/windows"
-    prefix.install "etc"
-    prefix.install "share"
+    libexec.install %w[bin etc libexec share]
+    rm_rf libexec/"bin/windows"
+
+    # Delete some lines to avoid the error like
+    # "cd: ../Cellar/confluent-platform/5.4.1/bin/../share/java: No such file or directory"
+    inreplace libexec/"bin/confluent-hub", <<~'EOS', ""
+      if [ -L /usr/local/bin/confluent-hub ]; then
```

Comment:
Could perhaps simplify this by making the `inreplace` just `if [ -L /usr/local/bin/confluent-hub ]; then` to `if false; then

Reply:
I've simplifed the logic in https://github.com/Homebrew/homebrew-core/pull/54858/commits/1393c44983af08b37b6ee1f66055c24574444cf0.